runtime/tr.source/trj9/optimizer/J9TransformUtil.cpp

```diff
@@ -1393,7 +1393,7 @@ TR::Node * J9::TransformUtil::calculateOffsetFromIndexInContiguousArray(TR::Comp
 
    if (shift)
       {
-      TR::Node *shiftNode = TR::Node::create(constOp, 0);
```

Comment:
Seems we can reduce the scope of `constOp` now and push it closer to its use.

Reply:
To me, it looks a bit funny if you do it that way, but this could just be the way that my brain likes repetition close together.